lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -248,6 +248,35 @@ function dosomething_northstar_save_id_field($uid, $northstar_response) {
 }
 
 /**
+ * Save the user's Northstar ID to their local profile.
+ *
+ * @param object $account
+ *   Fully loaded user object.
+ *
+ * @return string
+ *   The Northstar id or empty string on error.
+ */
+function dosomething_northstar_get_id($account = NULL) {
```

Comment:
hmm instead of putting this here it might make more sense to put this in the `dosomething_user` module as a [hander](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1264) for the `dosomething_user_get_field` function 
so that way you just have to call the function to get a field like normal and it returns this result alllways.